### DISCUSSION TITLE
allure reports integration with JMeter Function API testing

### DISCUSSION THREAD

**Discussion by rajukumar198912** [allure-framework/allure2#2758] [Ideas & Feedback]
I would like to integrate allure with  Jmeter function api testing.
To log resposen and request in report

**Comment by baev** [comment]
For the Allure Report, the integration isn't that straightforward. To make it work, you usually need to pre-aggregate the test results to make it useful. We have a working prototype, but since it requires additional services (we use influx) & specific JMeter configuration, we only offer it to the Allure TestOps users. If you are interested, you can contact the sales team at sales@qameta.io
